sql/src/main/java/io/crate/action/sql/DCLStatementDispatcher.java

```diff
@@ -44,21 +59,45 @@
 @Singleton
 public class DCLStatementDispatcher implements BiFunction<AnalyzedStatement, Row, CompletableFuture<Long>> {
 
-    private static final InnerVisitor INNER_VISITOR = new InnerVisitor();
+    private static class ExpressionParameterSubstitutionFormatter extends ExpressionFormatter.Formatter {
 
+        private final ParameterContext parameterContext;
+
+        ExpressionParameterSubstitutionFormatter(ParameterContext context) {
+            this.parameterContext = context;
+        }
+
+        @Override
+        public String visitParameterExpression(ParameterExpression node, Void context) {
+            String formattedExpression = null;
+            Symbol symbol = parameterContext.apply(node);
+            if (symbol != null) {
+                formattedExpression = symbol.representation();
```

Comment:
Would this work with arrays and objects? The `representation` method is for EXPLAIN - I don't think that we output a correct object literal string here.

Reply:
Seems to work yes

Reply:
However, do you have a better/more stable solution than this? The condition needs to also be human readable as it's displayed in the infromation_schema